ops/k8s/run.yaml

```diff
@@ -32,8 +32,14 @@ spec:
         - "-c"
         - |
           NODE_ID=${HOSTNAME##*-}
-          ./docker_command -b {{ boot_node }} --test-network-key-seed ${NODE_ID}
-
+          if [ ${NODE_ID} -eq 0 ]; then
+            ./docker_command --addr 0.0.0.0:3000 --test-network-key-seed ${NODE_ID}
+          else
+            sleep 30
+            BOOT_NODE_IP=$(host {{ boot_node_host }} | head -n 1 | cut -f4 -d ' ')
```

Comment:
In docker swarm running on AWS I found out that amount all Linux-provided tools that can resolve DNS that one that works reliably is `nslookup`, see: https://github.com/nearprotocol/pynear/blob/master/pynear/testnet/connection_tester.py#L9